suites/squid/nvmeof/tier-1_nvmeof_sanity.yaml

```diff
@@ -121,13 +122,16 @@ tests:
           - nqn: nqn.2016-06.io.spdk:cnode1
             serial: 1
             bdevs:
-              count: 1
-              size: 10G
-            listener_port: 5001
+              - count: 1
+                size: 10G
+                ns_create_image: true
+            listeners:
+              - node6
+            listener_port: 4420
             allow_host: "*"
         initiators:                             # Configure Initiators with all pre-req
           - subnqn: nqn.2016-06.io.spdk:cnode1
-            listener_port: 5001
+            listener_port: 4420
```

Comment:
You can move back to 5001 port in the PR. Since that works in pipeline without any issues. So no need to change it now.